nemo_skills/evaluation/metrics/utils.py

```diff
@@ -40,6 +40,12 @@ def is_correct_judgement(judgement, return_none=False) -> Union[bool, None]:
             return True
         elif verdict.lower().startswith("no"):
             return False
+    else:
```

Comment:
Could you maybe add a new metric type for now that will use this logic? We probably want to generalize things a bit in regards to judgement-only benchmarks, but for now I guess we can just support a custom logic just for this one and fix it properly later. I think your current change might introduce unintended consequences in other places relying on this function.

Reply:
resolved